src/components/WorkPoolCreateWizardStepInfrastructureConfiguration.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
Would be good to see this content in the localization module; if you put it there you have a convenient place to use markdown instead for the copy and then pass it to a markdown renderer here